.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        go-version: [1.17, 1.19, '1.20', 1.21]
```

Comment:
@fxamacker Did you intend to remove 1.21 here?

Reply:
@benluddy Good catch!  :+1:   Yes, I replaced 1.21 with 1.22 here with the intention of using 1.21 more extensively than the other versions but didn't finish refactoring workflows on Sunday.

I'll open issue so I don't forget.